pkg/asset/agent/common.go

```diff
@@ -100,7 +101,9 @@ func DetermineReleaseImageArch(pullSecret, pullSpec string) (string, error) {
 
 	releaseArch, err := ExecuteOC(pullSecret, getReleaseArch)
 	if err != nil {
-		logrus.Errorf("Release Image arch could not be found: %s", err)
+		if strings.Contains(err.Error(), "unable to read image") {
+			logrus.Debugf("Could not get release image to check architecture in a disconnected setup")
+		}
```

Comment:
nit: What do you think of just setting the log level to `debug` instead?
```suggestion
		logrus.Debugf("Release Image arch could not be found: %s", err)
```
I think the error message can be helpful for dev + other errors (i.e. not just ones with "unable to read image") is also logged. I guess the goal is to not mark it as an error when logging as it seems not affect any functionality?

Reply:
Oh okayy, sounds good! Thanks!!

Reply:
Thank you for the suggestion.